tests/dotnet-lib-6/Dockerfile.glibc

```diff
@@ -0,0 +1,22 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-20210924170333-538077f as builder
```

Comment:
Is there a specific need to use the Ubuntu 16.04 container to build?

Reply:
This one is recommended on .Net github repo, I couldn't find the exact page now sorry.

Reply:
Ok from https://github.com/dotnet/dotnet-buildtools-prereqs-docker Ubuntu 16.04 seems to still be maintained for now, and newer Ubuntu versions still follow the same image format.